src/main/java/io/logz/log4j2/LogzioAppender.java

```diff
@@ -370,6 +381,10 @@ private boolean validateQueueCapacity() {
             statusLogger.error("inMemoryQueueCapacityBytes should be a non zero integer or -1");
             return false;
         }
+        if (inMemoryLogsCountLimit <= 0 && inMemoryLogsCountLimit != DONT_LIMIT_QUEUE_SPACE) {
```

Comment:
`&&` should be `||` ?

Reply:
@asafm The only way for `inMemoryLogsCountLimit ` to be negative and valid is if it is equal to -1